app-backend/tile/src/main/scala/routes/HealthCheckRoute.scala

```diff
@@ -1,36 +1,32 @@
 package com.azavea.rf.tile.routes
 
-import com.azavea.rf.common.cache.kryo.KryoMemcachedClient
-import com.azavea.rf.database.{UserDao}
-import com.azavea.rf.database.util.RFTransactor
-
 import akka.http.scaladsl.model.HttpResponse
-import akka.http.scaladsl.server._
 import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server._
+import cats.effect.IO
+import com.azavea.rf.common.cache.kryo.KryoMemcachedClient
+import com.azavea.rf.database.UserDao
+import com.azavea.rf.database.util.RFTransactor
 import com.typesafe.scalalogging.LazyLogging
-import de.heikoseeberger.akkahttpcirce.ErrorAccumulatingCirceSupport._
-import io.circe.syntax._
-import doobie._
+import doobie.hikari.HikariTransactor
 import doobie.implicits._
-import doobie.postgres._
```

Comment:
Interacting with the endpoint should reveal if removing these will cause a problem, but worth holding off on merging this until we confirm the impact of the automatic type casts (Java/Scala types to PostgreSQL types) these imports were providing.

See: https://rollbar.com/Azavea/RasterFoundry/items/4815/